packages/react-sdk/src/withOptimizely.spec.tsx

```diff
@@ -192,4 +192,35 @@ describe('withOptimizely', () => {
       optimizelyReadyTimeout: 200,
     })
   })
+
+  it('should forward refs', () => {
+    interface FancyInputProps extends TestProps {
+      defaultValue: string
+    }
+    const FancyInput: React.RefForwardingComponent<HTMLInputElement, FancyInputProps> = (
+      props,
+      ref,
+    ) => <input ref={ref} className="fancyInput" defaultValue={props.defaultValue} />
+    const ForwardingFancyInput = React.forwardRef(FancyInput)
+    const OptimizelyInput = withOptimizely(ForwardingFancyInput)
+    const inputRef: React.RefObject<HTMLInputElement> = React.createRef()
+
+    const optimizelyMock: ReactSDKClient = ({
+      setUser: jest.fn(),
+    } as unknown) as ReactSDKClient
+
+    mount(
+      <OptimizelyProvider
+        optimizely={optimizelyMock}
+        timeout={200}
+        user={{ id: 'jordan' }}
+        userAttributes={{ plan_type: 'bronze' }}
+        isServerSide={true}
+      >
+        <OptimizelyInput ref={inputRef} defaultValue="hi" />
+      </OptimizelyProvider>,
+    )
+    expect(inputRef.current).toBeInstanceOf(HTMLInputElement)
+    expect(typeof inputRef.current!.focus).toBe('function')
```

Comment:
Can you also assert that `inputRef.current.value` is in fact the same reference as `component.find('input'). getDOMNode()`?